### PR TITLE
chore(release): release sdk 2.0.0b5

### PR DESCRIPTION
**Pull request by connor-mccarthy**
**Description of your changes:**
Includes a simplification/flattening of the release notes structure in `./sdk/RELEASE.md`.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title 
convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -119,13 +119,22 @@
...
+            'aliases': [],
+        },
+        {
+            'version': 'https://kubeflow-pipelines.readthedocs.io/en/2.0.0b5/',
+            'title': 'v2.0.0b5',
+            'aliases': [],
+        },
         {
             'version': 'https://kubeflow-pipelines.readthedocs.io/en/2.0.0b4/',
-            'title': 'v2.0.0b4 [pre-release]',
```
Comment (chensun) [inline docs/conf.py:128]: For my education what is the use of this label and why removing it?
Technically beta is still considered pre-release--pip install won't upgrade to a beta version unless add `--pre` flag.

File: sdk/RELEASE.md [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+
+## Features
+* Support `google.`-namespaced artifact types [\#8191](https://github.com/kubeflow/pipelines/pull/8191), [\#8232](https://github.com/kubeflow/pipelines/pull/8232), [\#8233](https://github.com/kubeflow/pipelines/pull/8233), [\#8279](https://github.com/kubeflow/pipelines/pull/8279)
+* Support dynamic importer metadata [\#7660](https://github.com/kubeflow/pipelines/pull/7660)
+
+## Breaking changes
 
 ## Deprecations
 
-## Bug Fixes and Other Changes
```
Comment (chensun) [inline sdk/RELEASE.md:13]: I would not replace "other" with "small" here. There could be large refactoring changes for example which are not "small" yet doesn't fit "Major features and improvements" or "Features"

Comment (connor-mccarthy) [reply sdk/RELEASE.md:13]: Agreed. Will revert this.

File: docs/conf.py (hunk 1 above)
Comment (connor-mccarthy) [reply docs/conf.py:128]: It's the label on the dropdown to select the version. This choice was to reduce verbosity from having several different pre versions all say `[pre-release]`, when this is implied by the `b` in the version.

Happy to re-add if you think being more explicit is better.

**Review by chensun — COMMENTED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**